documentation/modules/auxiliary/scanner/afp/afp_login.md

```diff
@@ -0,0 +1,41 @@
+## Vulnerable Application
+
+Apple Filing Protocol (AFP) is Apple's file sharing protocol similar to SMB, and NFS. This module attempts to bruteforce authentication credentials for AFP.
+
+References:
+[developer.apple.com](https://developer.apple.com/library/mac/documentation/Networking/Reference/AFP_Reference/Reference/reference.html)
+
+[developer.apple/com](https://developer.apple.com/library/mac/documentation/networking/conceptual/afp/AFPSecurity/AFPSecurity.html)
+
+1. `sudo apt-get install netatalk`
+2. edit `/etc/default/netatalk` and add the following lines:
+  ```
+  ATALKD_RUN=no
+  PAPD_RUN=no
+  CNID_METAD_RUN=yes
+  AFPD_RUN=yes
+  TIMELORD_RUN=no
+  A2BOOT_RUN=no
+```
+3. Restart the service: `sudo /etc/init.d/netatalk restart`
+
+## Verification Steps
+
+  1. Start msfconsole
+  2. Do: `use modules/auxiliary/scanner/afp/afp_login`
+  3. Do: `show options`
+      ...show and set options...
+  4. Do: `run`
+
+## Scenarios
+
```

Comment:
Can you add an L3 header saying what os/version of the software this is.
Example:
```
### netatalk v2.1 on Kali 2019.3
```